src/ios/PushPlugin.m

```diff
@@ -122,6 +161,27 @@ - (void)init:(CDVInvokedUrlCommand*)command;
      (UIRemoteNotificationTypeBadge | UIRemoteNotificationTypeSound | UIRemoteNotificationTypeAlert)];
 #endif
     
+    //  GCM options
+    [self setGcmSenderId: [NSString stringWithString: [iosOptions objectForKey:@"senderID"]]];
```

Comment:
@jtbdevelopment this will cause:

```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSPlaceholderString initWithString:]: nil argument'
```

if the `options.ios` object doesn't have a `senderID` object.